atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -119,6 +152,65 @@ def get_buildroot(self, worker_metadatas):
 
         return buildroots
 
+    def get_logs(self):
+        """
+        Build list of log files
+
+        :return: list, of log files
+        """
+
+        logs = None
+        output = []
+
+        # Collect logs from server
+        try:
+            logs = self.osbs.get_orchestrator_build_logs(self.build_id)
+        except OsbsException as ex:
+            self.log.error("unable to get build logs: %r", ex)
+            return output
+        except TypeError:
+            # Older osbs-client has no get_orchestrator_build_logs
+            self.log.error("Your osbs client doesn't support this functionality")
+            return output
+
+        self.log.info("logs=%s", logs)
+        platform_logs = {}
+        for entry in logs:
+            self.log.info("platform: %s, line: %s", entry.platform, entry.line)
+            platform_logs.setdefault(entry.platform, [])
+            platform_logs[entry.platform].append(entry.line)
+
+        for arch in platform_logs:
+            # Delete once closed
+            logfile = NamedTemporaryFile(prefix="%s-%s" % (self.build_id, arch),
+                                         suffix=".log", mode='wb')
+            for entry in platform_logs[arch]:
+                self.log.info("platform_logs[%s]: %s", arch, entry)
+                logfile.write(entry)
+            logfile.flush()
+            metadata = self.get_output_metadata(logfile.name, "%s.log" % arch)
+            output.append(Output(file=logfile, metadata=metadata))
+
+        logfile = NamedTemporaryFile(prefix="%s-orchestrator" % self.build_id,
+                                     suffix=".log", mode='wb')
+        orchestrator_logs = (log.line for log in logs if log.platform is None)
```

Comment:
Ah. Right. An earlier version of the code built an arches list that excluded None, so that was added for the None logs, but now we're building a list that includes None... Okay, I'll figure out something for that shortly here.